spec/features/publish/courses/editing_course_requirements_2025_spec.rb

```diff
@@ -1,55 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Don't we still need these tests, but without the year constraint? They should pass for any year now, right? 

Or do they exist elsehwere?

Reply:
All that this tested was that there was no link to "Other requirements" which we're dropping in this PR. 

Not sure it makes sense to test that something isn't there 🤔

Reply:
Nice one, that makes sense!